internal/authtoken/repository.go

```diff
@@ -262,21 +268,22 @@ func (r *Repository) ValidateToken(ctx context.Context, id, token string, opt ..
 	}
 
 	if err != nil {
-		return nil, fmt.Errorf("validate token: auth token: %s: %w", id, err)
+		return nil, errors.Wrap(err, op, errors.WithMsg(id))
 	}
 	return retAT, nil
 }
 
 // ListAuthTokens in an org and supports the WithLimit option.
 func (r *Repository) ListAuthTokens(ctx context.Context, withOrgId string, opt ...Option) ([]*AuthToken, error) {
+	const op = "authtoken.(Repository).ListAuthTokens"
 	if withOrgId == "" {
-		return nil, fmt.Errorf("list users: missing org id %w", errors.ErrInvalidParameter)
+		return nil, errors.New(errors.InvalidParameter, op, "missing org id")
```

Comment:
nice to see the cut/paste of the wrong context corrected.

Reply:
🙃  whoops